### DISCUSSION TITLE
Specifying training video paths in batch when using a new machine

### DISCUSSION THREAD

**Discussion by NeuTTH** [talmolab/sleap#1212] [Help!] [answered]
Hi! I am currently trying to add on a train model package with existing videos and models. I am prompted with the error message of not being able to find the videos, but there are a lot (100+) videos that I need to redefine the new file path one by one. Is there some ways in the CLI that enable SLEAP to use a new folder path for all videos?  Thanks!

**Comment by talmo** [comment, marked as answer]
Hey @NeuTTH,

Yep, you can do this programmatically to save a new SLP file by doing something like this:
```py
import sleap
labels = sleap.load_file("labels.v000.slp", search_paths=["path/to/video1.mp4", "path/to/video2.mp4", ...])
labels.save("labels.v000.fixed.slp")
```

You can also specify this in the CLI to `sleap-train`, but it's usually easier to save a new SLEAP file.

Note: If you're doing it via the GUI, a prompt will appear asking you to find the new paths to the video files. After locating the first one, it'll try to match the prefix to auto-detect the other ones as long as they have a shared parent folder at some point.

Let us know if that works for you!

Talmo

**Reply by NeuTTH** [reply]
Thanks!!
